client/js/src/main/scala/zio/zmx/client/frontend/views/ChartView.scala

```diff
@@ -151,7 +151,7 @@ object ChartView {
         ),
         // This is the place holder for a form that will allow us to manipulate the settings of the Chartview
         div(
-          cls := "w-1/2 h-full p-3 ml-2",
+          cls := "w-1/3 h-full p-3 ml-2",
           span(
             cls := "text-2xl font-bold",
             "Some Diagram info"
```

Comment:
This text should be gone with your change. For now we just want to see the config form.

Reply:
Looks way better, thanks for this